tests/unit/conftest.py

```diff
@@ -472,17 +471,14 @@ def license_record(db):
 def deposit(app, es, users, location):
     """New deposit with files."""
     record = dict(
-        metadata=dict(
-            upload_type='presentation',
-            title='Test title',
-            creators=[
-                dict(name='Doe, John', affiliation='Atlantis'),
-                dict(name='Smith, Jane', affiliation='Atlantis')
-            ],
-            description='Test Description',
-            publication_date='2013-05-08',
-            access_right='open'
-        )
+        title='Test title',
+        creators=[
+            dict(name='Doe, John', affiliation='Atlantis'),
+            dict(name='Smith, Jane', affiliation='Atlantis')
+        ],
+        description='Test Description',
+        publication_date='2013-05-08',
+        access_right='open'
```

Comment:
This was edited as this fixture was initiating Deposit with legacy metadata (this didn't pass the jsonschema validation)